src/main/java/pl/isa/freshmenindustries/game/Game.java

```diff
@@ -13,16 +13,18 @@ public String getName() {
         return name;
     }
 
-    public void setName(String name) {
+    public String setName(String name) {
```

Comment:
Przywróć to setName i setDescription do tego co było, czyli void i bez returna, bo nigdzie nie potrzebujesz zwracania wartości w setterze.